changelog/conv_parse_count.dd

```diff
@@ -0,0 +1,12 @@
+Return the number of characters consumed by `std.conv.parse`
+
+By setting the flag `doCount` to `Yes.doCount`, the function returns a named tuple. The tuple contains the fields `data` and `count`. The field `data` is of type `Target` and it contains the result of the original function. The field `count` is of type `size_t` and it contains the number of characters consumed.
```

Comment:
Sorry for nagging, but can you please keep the number of characters per line to a maximum of 120 chars / line?

Thanks

Reply:
Done! Thank you for the feedback.